models/nft/arbitrum/nft_arbitrum_transfers.sql

```diff
@@ -0,0 +1,85 @@
+{{ config(
+        alias ='transfers',
+        partition_by='block_date',
+        materialized='incremental',
+        file_format = 'delta',
+        post_hook='{{ expose_spells(\'["arbitrum"]\',
+                                    "sector",
+                                    "nft",
+                                    \'["hildobby"]\') }}',
+        unique_key = ['unique_transfer_id']
+)
+}}
+
+ SELECT t.evt_block_time AS block_time
+, date_trunc('day', t.evt_block_time) AS block_date
+, t.evt_block_number AS block_number
+, 'erc721' AS token_standard
+, 'single' AS transfer_type
+, t.evt_index
+, t.contract_address
+, t.tokenId AS token_id
+, 1 AS amount
+, t.from
+, t.to
+, t.evt_tx_hash AS tx_hash
+, 'arbitrum' || t.evt_tx_hash || '-erc721-' || t.contract_address || '-' || t.tokenId || '-' || t.from || '-' || t.to || '-' || '1' || '-' || t.evt_index AS unique_transfer_id
+FROM {{ source('erc721_arbitrum','evt_transfer') }} t
+    {% if is_incremental() %}
+        LEFT ANTI JOIN {{this}} anti
```

Comment:
For [test failure](https://github.com/duneanalytics/spellbook/actions/runs/3410941176/jobs/5674523685), you can retry with [ANTI JOIN](https://docs.databricks.com/sql/language-manual/sql-ref-syntax-qry-select-join.html).
```suggestion
        ANTI JOIN {{this}} anti
```

Reply:
Sure just changed it in a new commit, but why do you think that would work better? Here it worked fine with a `LEFT ANTI JOIN` but maybe I'm missing something: https://github.com/duneanalytics/spellbook/blob/main/models/addresses_events/ethereum/addresses_events_ethereum_first_funded_by.sql

Reply:
It seems like that didn't fix the issue 😔 Not sure what the problem is since I had this exact method worked on file linked above 🤷  Lmk if you have any ideas!

Reply:
It seems like the one you wrote is the first `ANTI JOIN` in the spellbook, and there is no other precedent to refer to at the moment. Maybe try this:
```
LEFT JOIN {{this}} anti
ON t.evt_tx_hash = anti.tx_hash
AND anti.tx_hash is NULL
```

Reply:
So I replace `anti` to `anti_table` and solve the syntax error. I think the word `anti` is a keyword in the databricks sql.